project/app/api/database.py

```diff
@@ -35,18 +16,71 @@
 
 class PostgreSQL:
     def __init__(self):
-
         self.DB_NAME = os.getenv("DB_NAME")
         self.DB_USER = os.getenv("DB_USER")
         self.DB_PASSWORD = os.getenv("DB_PASSWORD")
         self.DB_HOST = os.getenv("DB_HOST")
         self.DB_PORT = os.getenv("DB_PORT")
-        self.connection = psycopg2.connect(
-            dbname=self.DB_NAME, user=self.DB_USER, password=self.DB_PASSWORD, host=self.DB_HOST, port='5432')
 
-    def adapters(*args):
-        for adapter in args:
-            register_adapter(adapter, psycopg2._psycopg.AsIs)
+    connection = psycopg2.connect(dbname='bridges_to_prosperity_oct2018', user="postgres_labs28",
+                                  password='bridges28',
+                                  host="bridges-prosperity-labs28.cizmj9mbwva2.us-east-1.rds.amazonaws.com",
+                                  port=5432)
+    # psycopg2.connect(dbname=self.DB_NAME, user=self.DB_USER, password=self.DB_PASSWORD,
+    #                                host=self.DB_HOST, port='5432')
+
```

Comment:
The column names are capitalized here in this list, they are not in the data we are referencing.

Reply:
Yes this is a for the old data set, not the new onew